server/test-fault-tolerance.js

```
@@ -0,0 +1,379 @@
+      params: {
+        name: 'initialize_asr_got_graph',
+        arguments: {
+          task_description: '' // Empty string
+        }
+      }
+    };
+
+    try {
+      const response = await this.sendMCPRequest(request1);
+      
+      if (response.error && response.error.message.includes('task_description')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)